testing/src/main/scala/org/http4s/testing/ArbitraryInstances.scala

```diff
@@ -252,4 +255,102 @@ trait ArbitraryInstances {
       )
     } yield ServerSentEvent(data, event, id, retry))
   }
+
+  // https://tools.ietf.org/html/rfc2234#section-6
+  val hexDigit = oneOf(Seq('0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F'))
```

Comment:
I've started calling these `gen*`, since this could get imported who knows where now that it's public API.